week1/matrix_done.py

```diff
@@ -0,0 +1,87 @@
+class Matrix:
+    def __init__(self, list):
+        self.rows= list
+    def print(self):
```

Comment:
This is exactly what [`__str__`](https://docs.python.org/3/reference/datamodel.html#object.__str__) is for.

Please implement `__str__` instead.